src/app/auth/login/login.component.html

```diff
@@ -4,7 +4,6 @@
       <div class="error" > {{ loginFailure || '' }} </div>
       <div class="title_wrapper" routerLink="/home">
         <div class="auth-title">CLARK</div>
-        <div class="auth-title light">Cybersecurity Labs and Resource Knowledge-base</div>
```

Comment:
He wanted it to say "Cybersecurity Digital Library," not to remove it.  Also, I think this should be prevalent on all auth components (login, register, email verified, etc.), check with Dr. Kaza to be sure.

Reply:
Resolved with Resolve deletion